samples/istio/conformance/check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+# Ensure ITER8 environment variable is set
+if [[ -z ${ITER8} ]]; then
+    echo "ITER8 environment variable needs to be set to the root folder of Iter8"
+    exit 1
+else
+    echo "ITER8 is set to " $ITER8
+fi
+
+# Check if experiment has completed
+completed="Completed"
+stage=$(kubectl get experiment $EXPERIMENT -o json | jq -r .status.stage)
+if [[ $stage = $completed ]]; then
+    echo "Experiment has Completed"
+else
+    echo "Experiment must be $completed;" 
+    echo "Experiment is $stage;"
+    exit 1
+fi
+
+# Check if no packets have been lost by Fortio
+pod_name=$(kubectl get pods --selector=job-name=fortio -o jsonpath='{.items[*].metadata.name}')
+kubectl cp default/"$pod_name":shared/fortiooutput.json /tmp/fortiooutput.json -c busybox
+
+REQUESTSTOTAL=$(jq -r .DurationHistogram.Count /tmp/fortiooutput.json)
+REQUESTS200=$(jq -r '.RetCodes."200"' /tmp/fortiooutput.json)
+if [[ $REQUESTSTOTAL -eq $REQUESTS200 ]]; then
+    echo "Packets were not lost"
+else
+    echo "Packets were lost"
+    echo "total requests:" $REQUESTSTOTAL
+    echo "200 requests:" $REQUESTS200
+    exit 1
+fi
+
+# Check if versionRecommendedForPromotion is productpage-v1; implies winner found
+expectedVrfp="productpage-v1"
```

Comment:
Could we please replace this by a check for the winner instead?

No  matter what happens, vrfp will be the baseline in the conformance test (but winner will depend on whether the baseline satisfied SLOs or not).

Reply:
done